### ISSUE TITLE
SSH Connection issues

### ISSUE THREAD

**Issue by CodenvLover** [codenvy/codenvy#2661]
Hello,
I recently open an issue here about ssh-ing to CodeCommit, and it was solved, but Now I don't know what's going on with my workspace but it keeps rejecting ssh to my repo after an amount of time each time I change ssh key?it works for a moment but after that I need to change it again, it's a weird behavior that I can't understand!

Any suggestions?
Thank you

**Comment by ghost** [comment]
@CodenvLover most certainly IP address has changed

**Comment by CodenvLover** [comment]
it says Auth fails, and in the console it outputs **Not found ssh key uploader for git-codecommit.us-east-1.amazonaws.com**, all my ssh keys are already set

**Comment by ghost** [comment]
And it works when you generate a new key and save it to your codecommit account?

**Comment by CodenvLover** [comment]
No, now when I generate a new key ,it says auth fail or it takes so long and outputs connection time out

**Comment by ghost** [comment]
Probably IP address has changed

**Comment by CodenvLover** [comment]
is it from myside or CodeCommit server?

**Comment by ghost** [comment]
Must be CodeCommit.

**Comment by CodenvLover** [comment]
so what would be the solution ?
